tests/ordering/tests.py

```diff
@@ -405,6 +414,21 @@ def test_default_ordering_by_f_expression(self):
             attrgetter('headline')
         )
 
+    def test_order_by_aggregate(self):
+        self.assertQuerysetEqual(
+            OrderedByAggregateArticle.objects.order_by(Min('authors__id').asc()),
+            ['OArticle 2', 'OArticle 1'],
+            attrgetter('headline')
+        )
+
+    @ignore_warnings(category=RemovedInDjango31Warning)
```

Comment:
The implementation seems incorrect because I wouldn't expect this test to raise a deprecation warning.  "OrderedByAggregateArticle QuerySet won't use Meta.ordering in Django 3.1." -- which negates the entire purpose of this ticket! It might be that the ticket should be a wontfix as aggregations in querysets may have confusing side effects. Any input @charettes ?

Reply:
I think this feature would be hard to get right for the rare case where it's useful. It also go against our stance on trying to _disconnect_ `Meta.ordering` from aggregation as this PR demonstrates. I'd be in favour of won't fixing the ticket as well for these reasons.

It feels incoherent with the ongoing deprecation motivated by observed user confusion over the years from the interactions between these two features.